MatrixSDK/MXSession.m

```diff
@@ -748,9 +765,6 @@ - (void)serverSyncWithServerTimeout:(NSUInteger)serverTimeout
         // Check whether the caller wants to handle error himself
         if (failure)
         {
-            // Inform the app there is a problem with the connection to the homeserver
```

Comment:
Was it removed to avoid double changes of the state? One here and one by the caller?